shell/server/proxy.js

```diff
@@ -1616,8 +1633,8 @@ class Proxy {
 
       const mainUrl = process.env.ROOT_URL;
       const grainHost = PROTOCOL + "//" + request.headers.host;
-      response.setHeader("Content-Security-Policy", "frame-ancestors " + mainUrl + " " + grainHost);
-      response.setHeader("X-Frame-Options", "ALLOW-FROM " + mainUrl);
+      // response.setHeader("Content-Security-Policy", "frame-ancestors " + mainUrl + " " + grainHost);
```

Comment:
Sorry, I should've commented on the issue. This isn't ready for merge yet as it's blockking on this.